loleaflet/src/control/Control.JSDialogBuilder.js

```diff
@@ -1893,6 +1893,8 @@ L.Control.JSDialogBuilder = L.Control.extend({
 				var header = L.DomUtil.create('th', builder.options.cssClass, headers);
 				header.innerText = data.headers[h].text;
 			}
+		} else if (parentContainer.id == 'MacroSelectorDialog' && table.id == 'commands') {
+			L.DomUtil.addClass(table, 'empty');
```

Comment:
maybe better to make it for all treeviews?

Reply:
I mean add class empty always. and when you decide that it should be only for MacroSelectorDialog put it in the css selector.  (this will make this code generic, without exceptions for specific dialogs which makes it harder to maintain later)